tests/acceptance/README.md

```diff
@@ -0,0 +1,24 @@
+Acceptance tests
+================
+
+The tests will run on a file generated by the reporter.
+The file is generated using Mugshot, so you first need to run a browser:
+
+```sh
+npm run browser
+```
+
+Then you can generate the report. Two tests are made to fail.
```

Comment:
Negate the assertions in those tests so the suite won't fail. We need to add the acceptance suite in Travis and failures will stop the build. Also change the text here to something like 'two tests are made to produce diffs'.